documents/openapi/openapi.yaml

```diff
@@ -1149,7 +1132,61 @@ paths:
           $ref: '#/components/responses/NotFound'
         '500':
           $ref: '#/components/responses/InternalServerError'
-
+  /inbound-number:
+    get:
+      summary: Retrieve an inbound number
+      parameters: {} # TODO
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json: {} # TODO
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "500":
+          $ref: "#/components/responses/InternalServerError"
+        "502":
+          $ref: "#/components/responses/BadGateway"
+        "503":
+          $ref: "#/components/responses/ServiceUnavailable"
+    post:
+      summary: Create an inbound number
+      parameters: {} # TODO
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/InboundNumberCreateResponse"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "500":
+          $ref: "#/components/responses/InternalServerError"
+        "502":
+          $ref: "#/components/responses/BadGateway"
+        "503":
+          $ref: "#/components/responses/ServiceUnavailable"
+  /inbound-number/available:
+    get:
+      summary: Retrieve available inbound numbers
+      parameters: {} # TODO
+      responses: {} # TODO
+  /inbound-number/{inbound_number_id}:
+    post:
+      summary: Update an inbound number
+      parameters: {} # TODO
+      responses: {} # TODO
+  /inbound-number/{inbound_number_id}/off:  # TODO - Can this not be accomplished with the previous endpoint?
+    post:
+      summary: Set the inbound number as off
+      parameters: {} # TODO
+      responses: {} # TODO
+  /inbound-number/service/{service_id}:
+    get:
+      summary:  Get inbound numbers for a service
+      parameters: {} # TODO
+      responses: {} # TODO
```

Comment:
see comment above re TODO

Reply:
I deleted the parameter line and implemented a response.